dist/index.js

```diff
@@ -6592,6 +6592,14 @@ function run() {
         const captain = yield tc.downloadTool(url);
         core.debug('Installing to /usr/local/bin/captain');
         yield exec.exec('install', [captain, '/usr/local/bin/captain']);
+        const { stdout } = yield exec.getExecOutput('captain', ['--version'], {
+            silent: true
+        });
+        const cliVersion = stdout.replace('\n', '');
+        if (cliVersion !== version) {
```

Comment:
It'll be important that `captain --version` never changes it's format, like outputting `captain v0.0.0` instead of just `v0.0.0`. Otherwise, we'll break builds. Can we add a corresponding test in captain-cli that tests the format with a comment that we're expecting a specific format in the GitHub Action?

Reply:
I'll do that. I think changing that format would conceptually be a breaking change anyway!